docs/site/Setting-debug-strings.md

```diff
@@ -0,0 +1,329 @@
+---
+title: 'Setting debug strings'
+lang: en
+keywords: LoopBack 4.0, LoopBack 4, base-connector.js
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Setting-debug-strings.html
+summary:
+---
+
+You can specify debug strings when you run an application, as explained below,
+to display specific log output to the console. You can also redirect the output
+to a file, if desired. These techniques are often helpful in debugging
+applications.
+
+## Using debug strings
+
+LoopBack has a number of built-in debug strings to help with debugging.Specify a
```

Comment:
```suggestion
LoopBack has a number of built-in debug strings to help with debugging. Specify a
```

Reply:
oops, will fix it in https://github.com/strongloop/loopback-next/issues/4846